README.rst

```diff
@@ -1,27 +1,29 @@
 .. _cartridge-cli:
 
-===============================================================================
 Cartridge Command Line Interface
-===============================================================================
+================================
+
+.. image:: https://img.shields.io/github/v/release/tarantool/cartridge-cli?include_prereleases&label=Release&labelColor=2d3532
+   :alt: Cartridge CLI latest release on GitHub
+   :target: https://github.com/tarantool/cartridge-cli/releases
```

Comment:
The same badge is just below (but with 'Cartridge-CLI' instead of 'Cartridge CLI').